### PR TITLE
update tidb operator version descripiton

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update TiDB Operator overview to include v1.3.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-cn/pull/8398, https://github.com/pingcap/docs-cn/pull/8436
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
@qiancai Because there will be serious conflicts in v3.0~v5.3, I will cp from v5.3 to earlier versions.

File: tidb-operator-overview.md [hunk 1]
```diff
@@ -8,8 +8,6 @@ aliases: ['/docs/tidb-in-kubernetes/dev/']
 
 [TiDB Operator](https://github.com/pingcap/tidb-operator) is an automatic operation system for TiDB clusters in Kubernetes. It provides full life-cycle management for TiDB including deployment, upgrades, scaling, backup, fail-over, and configuration changes. With TiDB Operator, TiDB can run seamlessly in the Kubernetes clusters deployed on a public or private cloud.
 
-For the current TiDB release, the compatible version of TiDB Operator is v1.2.
-
 Currently, the TiDB Operator documentation (also named as TiDB in Kubernetes documentation) is independent of the TiDB documentation. To access the documentation, click the following link:
 
-- [TiDB in Kubernetes documentation](https://docs.pingcap.com/tidb-in-kubernetes/v1.2/).
\ No newline at end of file
+- [TiDB in Kubernetes documentation](https://docs.pingcap.com/tidb-in-kubernetes/stable/).
```
Comment (qiancai) [inline tidb-operator-overview.md:13]:
```suggestion
- [TiDB in Kubernetes documentation](https://docs.pingcap.com/tidb-in-kubernetes/stable/)
```

**Review by qiancai — COMMENTED** [review]
@ran-huang I've completed the review and left one comment. PTAL. Thanks.

**Review by qiancai — APPROVED** [review]

**Pull request merged**